packages/client/src/app/pages/forms/list/archived/archived.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react';
+import type { FormWithStats } from '@ff-client/queries/forms';
+import translate from '@ff-client/utils/translations';
+
+import { ArchivedItem } from '../archived-item/archived-item';
+
+import { ArchivedItems, Button, Wrapper } from './archived.styles';
+
+type Props = {
+  data: FormWithStats[];
+};
+
+export const Archived: React.FC<Props> = ({ data }) => {
+  const [isVisible, setIsVisible] = useState(false);
+  const toggleVisibility = (): void => setIsVisible(!isVisible);
+
+  const archived = data && data.filter(({ archived }) => archived);
+  const isEmpty = (data && !archived.length) ?? true;
+
+  if (isEmpty) {
+    return;
+  }
+
+  const label =
+    archived &&
+    (isVisible ? 'Hide ' : 'Show ') +
+      archived.length +
+      ' archived form' +
+      (archived.length === 1 ? '' : 's');
```

Comment:
We need to have these translatable. Lets go with plurals at all times in places where there may be more than 1 of items, Craft's translate isn't powerful enough for plurals I think.

```suggestion
  const label = translate('%s archived forms', isVisible ? 'Hide' : 'Show');
```